### PR TITLE
RELATED: RAIL-3560 Update workspace recordings

### PR DESCRIPTION
**Pull request by ivanmjartan**
---

Supported PR commands:

| Command                | Description            |
| ---------------------- | ---------------------- |
| `ok to test`           | Re-run standard checks |
| `extended test`        | BackstopJS tests       |
| `extended check sonar` | SonarQube tests        |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4624/ : SUCCESS in 6s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6248/ : FAILURE in 19m 10s

**Comment by ivanmjartan** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4639/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6267/ : SUCCESS in 17m 00s

**Comment by yenkins** [comment]
Link to Storybook test result: https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1093/artifact/libs/sdk-ui-tests/backstop/output/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/4640/ : SUCCESS in 7s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/1093/ : SUCCESS in 22m 04s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Review by pbenes — APPROVED** [review]

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by yenkins** [comment]
### Sonar scan result


More detail, see in https://sonarqube-gate.intgdc.com/dashboard?id=gooddata-ui-sdk-gate-PR1580

To scan for vulnerabilities in dependencies and run unit tests (to get coverage report in sonar) please comment your PR with 'extended check sonar'.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/1314/ : SUCCESS in 1m 11s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/1590/ : SUCCESS in 24s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-sonarqube-scan-zuul-gate/364/ : SUCCESS in 13m 58s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/6271/ : SUCCESS in 14m 28s

File: libs/sdk-ui-dashboard/src/model/commandHandlers/filterContext/dateFilter/tests/__snapshots__/changeDateFilterSelectionHandler.test.ts.snap [hunk 1]
```diff
@@ -16,6 +16,12 @@ Array [
 exports[`changeDateFilterSelectionHandler should set the date filter in the state when a date filter is changed 1`] = `
 Object {
   "dateFilter": Object {
+    "dateFilter": Object {
```
Comment (no23reason) [inline libs/sdk-ui-dashboard/src/model/commandHandlers/filterContext/dateFilter/tests/__snapshots__/changeDateFilterSelectionHandler.test.ts.snap:19]: This is extremely suspicious!

Comment (pbenes) [reply libs/sdk-ui-dashboard/src/model/commandHandlers/filterContext/dateFilter/tests/__snapshots__/changeDateFilterSelectionHandler.test.ts.snap:19]: Sorry I missed that in the review, should we revert the pull-request?.

**Pull request merged**